ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -1432,6 +1459,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-workers-marketimage
+- as: azure-ipi-workers-marketimage-p3-f28-long-duration-cloud
+  cron: 22 23 11 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      E2E_RUN_TAGS: '@azure-ipi and @network-ovnkubernetes and not @fips and @long-duration'
+      TAG_VERSION: '@4.12'
+      TEST_SCENARIOS: Cluster_Infrastructure
+      TEST_TIMEOUT: "90"
+    test:
+    - chain: openshift-e2e-test-qe-long-duration
+    workflow: cucushift-installer-rehearse-azure-ipi-workers-marketimage
```

Comment:
This one is same with 4.12, can we change it to `cucushift-installer-rehearse-azure-mag-ipi-fullyprivate` or remove it only keep one in arm.

Reply:
done